sources/advanced/Cli.ts

```diff
@@ -1,10 +1,10 @@
+import * as platform                                            from 'clipanion/platform';
```

Comment:
This change shouldn't be needed.

Reply:
This is the node's native way, compatible with the whole ecosystem, including Node, TS, and modern bundlers